tests/simple_nonlinear/screen-output

```diff
@@ -0,0 +1,176 @@
+-----------------------------------------------------------------------------
+-----------------------------------------------------------------------------
+
+Loading shared library <./libsimple_nonlinear.so>
+
+Test for p = -1000
+zerozero 0: Finite difference = 3.4588e+30. Analytical derivative = 3.4588e+30
+zerozero 1: Finite difference = 9.45819e+27. Analytical derivative = 9.45833e+27
+zerozero 2: Finite difference = 1.15028e+28. Analytical derivative = 1.15029e+28
+zerozero 3: Finite difference = -1.31479e+37. Analytical derivative = -0
+   Error: The derivative of the viscosity to the strain rate is too different from the analitical value.
+zerozero 4: Finite difference = -1.4336e+22. Analytical derivative = -0
+   Error: The derivative of the viscosity to the strain rate is too different from the analitical value.
+onezero 0: Finite difference = -7.68623e+29. Analytical derivative = -7.68623e+29
+onezero 1: Finite difference = 5.16403e+29. Analytical derivative = 5.16403e+29
+onezero 2: Finite difference = -2.09143e+29. Analytical derivative = -2.09143e+29
+onezero 3: Finite difference = -6.89496e+44. Analytical derivative = -6.89496e+44
+onezero 4: Finite difference = -4.52403e+29. Analytical derivative = -4.52403e+29
+oneone 0: Finite difference = -3.4588e+30. Analytical derivative = -3.4588e+30
+oneone 1: Finite difference = -9.45857e+27. Analytical derivative = -9.45833e+27
+oneone 2: Finite difference = -1.1503e+28. Analytical derivative = -1.15029e+28
+oneone 3: Finite difference = -1.31479e+37. Analytical derivative = -0
+   Error: The derivative of the viscosity to the strain rate is too different from the analitical value.
+oneone 4: Finite difference = -1.4336e+22. Analytical derivative = -0
+   Error: The derivative of the viscosity to the strain rate is too different from the analitical value.
```

Comment:
So it looks like a whole lot of the tests fail by a large amount. Do they pass on your machine?